edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.IoTHub/stream/StreamRequestListener.cs

```diff
@@ -38,62 +38,63 @@ public void Dispose()
             // Don't dispose the cts here as it can cause an ObjectDisposedException.
         }
 
-        async Task Pump(IModuleClient moduleClient)
+        Task Pump(IModuleClient moduleClient)
         {
-            while (!this.cts.IsCancellationRequested)
-            {
-                try
-                {
-                    await this.streamLock.WaitAsync(this.cts.Token);
-                    await this.ProcessStreamRequest(moduleClient);
-                }
-                catch (Exception e)
-                {
-                    Events.ErrorInPump(e);
-                }
-            }
+            return Task.CompletedTask;
```

Comment:
This function only gets called once? I'm assuming exiting immediately shouldn't cause any harm.

Reply:
Yes, we just create a task initially and then expect it to keep processing streams.

Reply:
This should not even be called, unless someone sets EnableStreams to true explicitly (it is false by default)